.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,30 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: rocq-*.opam
+
+      - name: 🐫🐪🐫 Get Rocq dependencies
+        run: opam install --deps-only ./rocq-*.opam
```

Comment:
this won't work well since stdlib frm the stdlib repo is dep of rocq-prover which is in this repo

Reply:
Ups. What should we do? Should we exclude rocq-prover from the job?

Reply:
use the same package list as in build template I guess
>PKGS=rocq-runtime,coq-core,rocq-core,coqide-server,rocq-devtools

Reply:
(+ rocqide if you want)

Reply:
Ok let's do that, tho it is a pity we don't have a central place, or we cannot do just yet "*.opam"

I will add rocqide , but I want to get things running first without GTK to understand the impact GTK has on the build times.

[I'm starting to think having rocq-prover.opam in this repos is a bug, but that's offtopic here]